### DISCUSSION TITLE
database

### DISCUSSION THREAD

**Discussion by avrahamyosuf** [symfony/symfony#45879] [Q&A]
guys i need some help. I had a problem with my alphabetical order on my website. Since I added later, the letter A comes to the bottom, I need an idea to fix it.As in the picture, the position of the letter D needs to be changed.
![image](https://user-images.githubusercontent.com/86631841/160576558-140be7e2-07c5-49c3-985d-18a2048a651c.png)

**Comment by excitedbox** [comment]
add a sort to the query? If you don't want alphabetical you will need to add a order column and sort by that.
